CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to the "vscode-java-dependency" extension will be documented
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.17.0
+### Added
+- Add contextual title and icon for `Java Project` explorer. [PR#423](https://github.com/microsoft/vscode-java-dependency/pull/423)
+- Allow adding library folder in `Java Project` explorer for unmanaged folders. [PR#434](https://github.com/microsoft/vscode-java-dependency/pull/434)
```

Comment:
Allow adding library folders into the `Referenced Libraries` on Windows and Linux (Press `Alt` to toggle out the button).